src/Controller/PostfinanceResponseController.php

```diff
@@ -34,17 +37,26 @@ public function processAcceptResponse(Request $request, PaymentInterface $paymen
     $plugin_definition = $payment->getPaymentMethod()->getPluginDefinition();
 
     // Generate local SHASign.
-    $request_values = $request->query->all();
+    $request_values = simplexml_load_string($request->query->get('data'));
```

Comment:
A but surprised that we are changing how this works here, I would assume this has worked before? Do we have a failing test with a wrong sha here somewhere?